de-webapp/src/main/java/org/iplantc/de/server/rpc/GwtRpcController.java

```diff
@@ -67,6 +67,9 @@ public String processCall(String payload) throws SerializationException {
 
             RPCRequest rpcRequest = RPC.decodeRequest(payload,
                                                       this.remoteServiceClass);
+            if (remoteService instanceof DeGwtRemoteLoggingServiceImpl) {
+                ((DeGwtRemoteLoggingServiceImpl)remoteService).setRequest(getThreadLocalRequest());
```

Comment:
I was looking at this again and realized we're already passing the request during `handleRequest` as long as the RemoteService implements HasHttpServletRequest.  I went ahead and had `DeGwtRemoteLoggingServiceImpl` do that which removed the need to have this separate bit of logic.